release-process.md

```diff
@@ -0,0 +1,106 @@
+# Release process
+
+This document describes our release process for open-source projects.
+
+## Contributors
+
+### New features, changes, and deprecations
+
+When adding new features (i.e., functions or classes), docstrings should properly
+indicate in which version the feature first appeared using the `.. versionadded::`
+directive.
+
+When changing the API,` .. versionchanged::` should be used. And deprecations should
+be indicated with `.. deprecated::`. Read
+[ploomber-core docs](https://ploomber-core.readthedocs.io/en/latest/deprecation.html)
+for details.
+
+
+### CHANGELOG
+
+Each repository contains a `CHANGELOG` file in the root directory. Each PR should
+contain a list of items, so we keep it up-to-date. Note that the `CHANGELOG` targets
+end-users (while `git log` targets the Ploomber development team); this implies
+that there might be changes that we don't include in the `CHANGELOG`, but they exist
+in the `git log`, for example changes to the CI configuration, new tests
+added/fixed.
+
+These are changes that we add to the `CHANGELOG`:
+
+- [Doc] Important documentation changes
+- [Feature] New features
+- [API Change] API breaking changes
+- [Fix] Bug fixes
```

Comment:
Is this the common convention?

Reply:
yeah, loosely based on this: https://scikit-learn.org/stable/whats_new/v1.2.html